scio-core/src/main/scala/com/spotify/scio/transforms/ScalaFutureHandlers.scala

```diff
@@ -27,7 +27,7 @@ import scala.concurrent.duration.Duration
 /** A [[FutureHandlers.Base]] implementation for Scala [[Future]]. */
 trait ScalaFutureHandlers[T] extends FutureHandlers.Base[Future[T], T] {
   @transient
-  private lazy implicit val immediateExecutionContext = new ExecutionContext {
+  implicit private lazy val immediateExecutionContext = new ExecutionContext {
```

Comment:
This also feels slightly counter intuitive since we usually say "implicit val" and I often grep for that exact match. 🤷‍♂

Reply:
access modifier aside... I would say `implicit lazy val` is slightly more intuitive 😄 and it's also inline with scalac style. It's because of these preferences/inconsistencies that i want smth consistent.